src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,19 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of validating files package-info.java and MyClass.java:
+ * </p>
+ * <p>package-info.java</p>
```

Comment:
Please add a little bit more than the file name so it is clear this is what you are referencing before the actual code.

I would combine `Examples of ...` with the file name for each example. Example: `Example of validating XXX file:`.
If some file is used more than once, say what makes the difference like "fixing violation".

Reply:
Done